### PR TITLE
Objective update

### PR DESCRIPTION
**Pull request by NickAragua**
- Fixed an issue preventing reinforcement of scenarios with forces already on the strategic board.
- Fixed an issue causing "squished" rendering for the strategic board when loading saves in some states (by getting rid of the grid bag layout and using a more rigid grid layout).
- Bulletproofing

### CODE REVIEW

File: MekHQ/src/mekhq/gui/StratconPanel.java [hunk 1]
```diff
@@ -117,7 +117,10 @@
      * Constructs a StratconPanel instance, given a parent campaign GUI and a pointer to an info area.
      */
     public StratconPanel(CampaignGUI gui, JLabel infoArea) {
-        campaign = gui.getCampaign();
+        if (gui != null) {
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/gui/StratconPanel.java:120]: How does the rest of this survive without a `Campaign` instance? I see lots of other places where the code would die.

**Review by sixlettervariables — COMMENTED** [review]
One question.

File: MekHQ/src/mekhq/gui/StratconPanel.java (hunk 1 above)
Comment (NickAragua) [reply MekHQ/src/mekhq/gui/StratconPanel.java:120]: To be honest, it's probably unnecessary. I wanted to re-test the StratconTab layout without having to reload the game, so I put in a call to InitTab in a couple of places, which re-initializes the StratconPanel - including once before the campaign GUI is loaded.

Comment (NickAragua) [reply MekHQ/src/mekhq/gui/StratconPanel.java]: Reverted.

**Review by Windchild292 — APPROVED** [review]

**Pull request merged**